### PR TITLE
Update IP detection routines

### PR DESCRIPTION
**Pull request by bexelbie**
They now match vagrant service-manager

### CODE REVIEW

**Comment by bexelbie** [comment]
@praveenkumar please review and test

**Comment by bexelbie** [comment]
Goes with vagrant-service-manager PR https://github.com/projectatomic/vagrant-service-manager/pull/148

File: adb-utils.spec [hunk 1]
```diff
@@ -44,6 +44,9 @@ ln -s /opt/adb/add_insecure_registry %{buildroot}%{_bindir}/add_insecure_registr
 %doc LICENSE  README.rst
 
 %changelog
+* Tue Apr 05 2016 Brian Exelbierd <bex@pobox.com> 1.4-2
```
Comment (navidshaikh) [inline adb-utils.spec]: The version needs to be 1.5-1 , since it is a code change.

Comment (navidshaikh) [inline adb-utils.spec]: Line no:2  of SPEC file also needs to be updated with version bump as

```
 Version:       1.5
```

**Comment by navidshaikh** [comment]
Cool! LGTM

**Pull request merged**